athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/optimizations/pushdown/HintsSubtype.java

```diff
@@ -0,0 +1,40 @@
+/*-
+ * #%L
+ * Amazon Athena Query Federation SDK
+ * %%
+ * Copyright (C) 2019 - 2022 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connector.lambda.metadata.optimizations.pushdown;
+
+public enum HintsSubtype
+    implements PushdownSubTypes
+{
+    //This indicates that the connector does not support the same default collate setting of Athena
+    NON_DEFAULT_COLLATE("non_default_collate");
```

Comment:
This basically indicate that this connector is non-default; meaning it doesn't match Athena's engine default sorting

Reply:
alternative name; `UNMATCHED_COLLATE` or `NON_MATHCING_COLLATE`